pkg/mapper/mapper.go

```diff
@@ -77,12 +100,16 @@ func (m *MetricMapper) InitFromYAMLString(fileContents string, cacheSize int, op
 		return err
 	}
 
-	if n.Defaults.Buckets == nil || len(n.Defaults.Buckets) == 0 {
-		n.Defaults.Buckets = prometheus.DefBuckets
+	if n.Defaults.HistogramOptions == nil {
```

Comment:
I thought about moving this into the mapper_defaults file, as setting the defaults behavior seems like it could go there (if someone uses the defaults without this, it would be weird). Open to suggestions